packages/expo-ui/src/swift-ui-primitives/Image.tsx

```diff
@@ -0,0 +1,49 @@
+import { requireNativeView } from 'expo';
+
+import { ViewEvent } from '../types';
+import { type CommonViewModifierProps } from './types';
+
+export interface ImageProps extends CommonViewModifierProps {
+  /**
+   * The name of the system image (SF Symbol).
+   * For example: 'photo', 'heart.fill', 'star.circle'
+   */
+  systemName: string;
+  /**
+   * The size of the system image.
+   */
+  size?: number;
+  /**
+   * The color of the system image.
+   * Can be a color name like 'red', 'blue', etc.
```

Comment:
```suggestion
   * Can be a color name like '#ff00ff', 'red', 'blue', etc.
```
Just to make it clear we're also accepting hex colors.